package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-starter",
-  "version": "2.2.0",
+  "version": "2.4.0",
```

Comment:
2.3.1 was a thing but got over-written in a merge somewhere... Thus the jump to 2.4